components/my-account/User/UserTable/UserTable.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import { composeStories } from '@storybook/testing-react'
+import { render } from '@testing-library/react'
+
+import * as stories from './UserTable.stories' // import all stories from the stories file
+
+const { UserTableComponent } = composeStories(stories)
+
+describe('[component] - KiboDataTable', () => {
+  it('should render table', () => {
+    render(<UserTableComponent />)
+  })
+})
```

Comment:
Mock KiboDataTable, add following unit testcases
  should render component - check acutal passed rows are rendered or not
  should edit user when user clicks on edit button
  should show delete confirmation dialog when user clicks on delete icon